web500/templates/login.html

```diff
@@ -3,14 +3,14 @@
 {% block content %}
   <h1>welcome to web500</h1>
   {% if invalid_login %}
-  <p>Username is taken</p>
+  <p>Nickname is taken or is too long (max: 15 chars).</p>
```

Comment:
A little bit lazy :P but we can fix this up later with the change to per-room usernames.